docs/README.md

```diff
@@ -1,20 +1,56 @@
 ## Generate docs automatically
 
-Run
+To generate documentation for `Games.jl`, run
+
 ```
 $julia make.jl
 ```
 
-The generated docs can be accessed in `build/index.html`.
+The generated docs can be accessed in `docs/build/index.html`.
+
+It includes three parts to generate the documentation.
```

Comment:
Not sure what "it" refers to here -- I'm guessing the process of generating documentation?
If so:
The process of generating documentation includes three parts.